tools/tcpdrop.py

```diff
@@ -213,55 +219,179 @@
 else:
     bpf_text = bpf_text.replace('FILTER_NETNS', '')
 
+# the reasons of skb drop
+drop_reasons = {
+    0: "SKB_NOT_DROPPED_YET",
+    1: "SKB_CONSUMED",
+    2: "SKB_DROP_REASON_NOT_SPECIFIED",
```

Comment:
May we can omit the `SKB_DROP_REASIN_` prefix.

Reply:
Good catch! 

I was thinking the same, so let's remove the prefix from that ;)

Reply:
Done.